src/Controller/MainController.php

```diff
@@ -37,13 +37,35 @@ public function __construct(\Twig_Environment $twig, EntityRepository $threadRep
      */
     public function indexAction(Request $request)
     {
-        $response = new Response();
-        $threads = $this->threadRepository->findAll();
+
+
+        //$allThreads = count( $this->threadRepository -> findAll());
+        $currentPage = $request->get('page', 1);
+
+        //var_dump(($currentPage - 1) * 10); die;
```

Comment:
would be worth a little tidying up. getting rid of these commented lines and excess empty lines. Empty lines are fine for spacing and making the code readable but no need for more than one empty line in a row.